omnibus_overrides.rb

```diff
@@ -1,16 +1,12 @@
 build_iteration 1
 
-# Using pins that agree with chef 13.0.118.
```

Comment:
do we want to pull in any of this stuff? These are only for the omnibus builds and all we need is the gemspec bit

Reply:
Yeah, that slipped in because of some git related follies (I discovered I needed a few of the maintenance PRs to build/test). Fixing.